bot/exts/utils/utils.py

```diff
@@ -50,71 +55,11 @@ def __init__(self, bot: Bot):
 
         self.base_pep_url = "http://www.python.org/dev/peps/pep-"
         self.base_github_pep_url = "https://raw.githubusercontent.com/python/peps/master/pep-"
+        self.peps_listing_api_url = "https://api.github.com/repos/python/peps/contents?ref=master"
```

Comment:
Making these class constants would help in making them easier to track of in case anything changes

Reply:
Addressed in 86f2024